tests/unit_tests/testnavigator.h

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "helper.h"
+
+class SettingsHolder;
+
+class TestNavigator final : public TestHelper {
+  Q_OBJECT
+
+ private slots:
+  void init();
+
+  void navbar_button_telemetry();
```

Comment:
nit:
```suggestion
  void testNavbarButtonTelemetry();
```

I know this is not standardized right now... So feel free to ignore.